tests/unit/macros/materializations/incremental/test_convert_to_list_macro.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from tests.unit.macros.base import MacroTestBase
+
+
+class TestConvertToListMacro(MacroTestBase):
+    @pytest.fixture(scope="class")
+    def macro_folders_to_load(self) -> list:
+        return ["macros/materializations/incremental"]
+
+    @pytest.fixture(scope="class")
+    def template_name(self) -> str:
+        return "convert_to_list.sql"
+
+    @pytest.mark.parametrize(
+        "func_input,expected", [(["a"], ["a"]), (None, []), ([], [])]
```

Comment:
Recommend adding a string -> list test case.

Reply:
Sure, I added that one.